site/content/docs/5.2/components/modal.md

```diff
@@ -554,6 +575,13 @@ exampleModal.addEventListener('show.bs.modal', event => {
 
 Toggle between multiple modals with some clever placement of the `data-bs-target` and `data-bs-toggle` attributes. For example, you could toggle a password reset modal from within an already open sign in modal. **Please note multiple modals cannot be open at the same time**—this method simply toggles between two separate modals.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant can not be used because it does not respect the Orange Design System specifications.
```

Comment:
For usability reasons. Toggling between modals is not something to be encouraged.

Reply:
New message:
This toggle behavior between multiple modals should not be used because it does not respect the Orange Design System specifications. It is not recommended for usability reasons.

For the correct design, please refer to [Modals](https://system.design.orange.com/0c1af118d/p/16d9f3-modals/b/774d3d) on the Orange Design System website.